### DISCUSSION TITLE
Login name casing

### DISCUSSION THREAD

**Discussion by DavideWalder** [zitadel/zitadel#4732] [Q&A] [answered]
A major complaint from our users is the case sensitive login. The local part of email addresses is case sensitive, but it is recommended not to treat it as such (to avoid having two users `john.doe` and `John.Doe` which might be able to access each others resources). The main issue is that when a user enters its login name with a different case, it is rejected (the same happens for external IDP users, even though the external IDP recognises them).

Proposition: make login names case insensitive (preventing the creation of duplicates)

**Comment by hifabienne** [comment, marked as answer]
Hi @DavideWalder 

Thanks for reporting.
I have created an issue for this.
https://github.com/zitadel/zitadel/issues/4840
